src/util/fields.py

```diff
@@ -1121,3 +1138,48 @@ def update_server_actions_fields(cr, src_model, dst_model=None, fields_mapping=N
         ) % {"src_model": src_model, "dst_model": dst_model, "actions": ", ".join(action_names)}
 
         add_to_migration_reports(message=msg, category="Server Actions")
+
+
+def update_custom_filter(cr, old, model, new=None):
+    arch_db = (
+        get_value_or_en_translation(cr, "ir_ui_view", "arch_db")
+        if column_exists(cr, "ir_ui_view", "arch_db")
+        else "arch"
+    )
+    match_old = r"\y{}\y".format(re.escape(old))
+    cr.execute(
+        """
+            SELECT id
+              FROM ir_ui_view
+             WHERE {} ~ %s
+               AND type = 'search'
+               AND model = %s
+               AND active = 't'
+        """.format(
+            arch_db
+        ),
+        [match_old, model],
+    )
+    for view_id in cr.fetchall():
+        try:
+            with suppress(_Skip), edit_view(cr, view_id=view_id, active=None) as view:
+                for elem in view.xpath("//filter[contains(@context, '{0}')]".format(old)):
```

Comment:
Should be moved around here https://github.com/odoo/upgrade-util/blob/3c7205a11504e0e018ca48ca6d0df65a2152f7ce/src/util/domains.py#L338

Reply:
Yes, don't move it to `domains.py`.